File: app/src/main/java/com/neome/java/api/meta/base/dto/EntVdSendEmail.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdSendEmail extends EntVdAutoStepWithError
{
  @Nullable
  public StudioDtoArgValueParameter attachmentField;

  @Nullable
  public StudioDtoUserFilter bccUsers;

  @Nullable
  public StudioDtoUserFilter ccUsers;

  @Nullable
  public StudioValueParagraph message;

  @Nullable
  public StudioDtoUserFilter replyToUsers;

  @Nullable
  public StudioValueText subject;

  @Nullable
  public StudioDtoUserFilter toUsers;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdSendEmail.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStepWithError
import com.neome.api.meta.base.dto.StudioDtoArgValueParameter
import com.neome.api.meta.base.dto.StudioDtoUserFilter
import com.neome.api.meta.base.dto.StudioValueParagraph
import com.neome.api.meta.base.dto.StudioValueText

interface EntVdSendEmail : EntVdAutoStepWithError
{
  val attachmentField: StudioDtoArgValueParameter?
  val bccUsers: StudioDtoUserFilter?
  val ccUsers: StudioDtoUserFilter?
  val message: StudioValueParagraph?
  val replyToUsers: StudioDtoUserFilter?
  val subject: StudioValueText?
  val toUsers: StudioDtoUserFilter?
}